ui/src/components/SelectNSFromItem/index.tsx

```diff
@@ -25,28 +29,36 @@ export default function SelectNSFromItem({
     return (
       <div>
         {menu}
-        <Divider style={{ margin: '10px 0' }} />
-        <div
-          onClick={() => setVisible(true)}
-          style={{ padding: '8px', cursor: 'pointer' }}
-        >
-          <PlusOutlined />
-          <span style={{ marginLeft: '6px' }}>
-            {intl.formatMessage({
-              id: 'OBDashboard.Cluster.New.BasicInfo.AddNamespace',
-              defaultMessage: '新增命名空间',
-            })}
-          </span>
-        </div>
+        {hasSysAccess ? (
```

Comment:
`system:write` is required to create a namespace. The above `access.systemread || access systemwrite` is incorrect.